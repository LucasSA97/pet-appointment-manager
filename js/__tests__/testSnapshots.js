const cliente = {
    nombre: 'Lucas',
    balance: 50,
    tipo: 'Normal'
}

describe('Test al cliente', () => {
    test('Es Lucas', () => { 
        expect(cliente).toMatchSnapshot()
     })
})

//npm t -- -u Actualizar√° el snapshot