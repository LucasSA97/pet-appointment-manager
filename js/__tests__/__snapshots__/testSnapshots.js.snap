// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test al cliente Es Lucas 1`] = `
{
  "balance": 50,
  "nombre": "Lucas",
  "tipo": "Normal",
}
`;
